RUFAS/biophysical/animal/digestive_system/enteric_methane_calculator.py

```diff
@@ -0,0 +1,331 @@
+        is_lactating: bool
+            Indicator of cow's lactating status.
+        milk_fat: float
+            Milk fat (from animal input), % of milk.
+        metabolizable_energy_intake: float
+            Metabolizable energy intake, Mcal/kg dry matter.
+        nutrient_amounts: Dict[str, float]
+            Amounts of nutrients in pen ration, calculated per animal, see Notes section for units.
+        nutrient_concentrations: Dict[str, float]
+             Concentrations of nutrients in pen ration, calculated per animal, percentages.
+        methane_mitigation_method: str
+            The name of the methane mitigation feed additives. The accepted names are
+                '3-NOP', 'Monensin', 'Essential Oils', and 'Seaweed'.
+        methane_mitigation_additive_amount: float
+            The dosage of the feed additive, mg/kg DMI.
+
+        Returns
+        -------
+        float
+            The daily enteric emissions for cows (g/day).
+
+            Notes
+            -----
+            nutrient_amount_units = {
+                "dm": "kg/animal",
+                "CP": "percent of DM",
+                "ADF": "percent of DM",
+                "NDF": "percent of DM",
+                "lignin": "percent of DM",
+                "ash": "percent of DM",
+                "phosphorus": "percent of DM",
+                "potassium": "percent of DM",
+                "N": "percent of DM",
+            }
+
+        """
+        dry_matter_intake = nutrient_amounts["dm"]
+        NDF_concentration = nutrient_concentrations["NDF"]
+        EE_concentration = nutrient_concentrations["EE"]
+        starch_concentration = nutrient_concentrations["starch"]
+
+        if is_lactating:
+            methane_emission = EntericMethaneCalculator._lactating_cow_manure(
+                body_weight,
+                milk_fat,
+                metabolizable_energy_intake,
+                nutrient_amounts,
+                nutrient_concentrations,
+                methane_model,
+            )
+        else:
+            methane_emission = EntericMethaneCalculator._dry_cow_manure(
+                methane_model, metabolizable_energy_intake, nutrient_amounts, nutrient_concentrations
+            )
+
+        if methane_mitigation_method:
+            methane_yield = 0.0
+            methane_yield_reduction = 0.0
+            if dry_matter_intake != 0:
+                methane_yield = methane_emission / dry_matter_intake
+                methane_yield_reduction = MethaneMitigationCalculator.methane_mitigation(
+                    NDF_concentration,
+                    EE_concentration,
+                    starch_concentration,
+                    methane_mitigation_method,
+                    methane_mitigation_additive_amount,
+                )
+
+            methane_emission = (
+                methane_yield
+                * (1 + methane_yield_reduction / GeneralConstants.FRACTION_TO_PERCENTAGE)
```

Comment:
```suggestion
                * (1 + methane_yield_reduction * GeneralConstants.PERCENTAGE_TO_FRACTION)
```
All of the constants in `GeneralConstants` are/should be multiplicative